src/main/java/org/traccar/protocol/NavtelecomProtocolDecoder.java

```diff
@@ -295,6 +295,84 @@ protected Object decode(
                                     value = buf.readByte();
                                     position.set(Position.PREFIX_TEMP + (j + 2 - 45), (value != 0x80) ? value : null);
                                     break;
+                                case 53:
+                                    value = buf.readUnsignedShortLE();
+                                    if (value == 0x7FFF) {
+                                        position.set(Position.KEY_FUEL_LEVEL + "Level", (Boolean) null);
+                                        position.set(Position.KEY_FUEL_LEVEL, (Boolean) null);
```

Comment:
What is this for?

Reply:
If the value is 0x7FFF, this is an error code, i.e.  the device does not receive information from the CAN bus on fuel.  It is necessary that the server does not process this value.  Neither as a parameter with a percentage value, nor as a parameter with liters.

Reply:
I understand that, but why set it to null?

Reply:
Maybe I didn't understand how to do it right.  The plan was such that the system would not display fuel values ​​from this message... put null purely by analogy with previous error handling, for example, field #44

Reply:
Ok, got it, thanks for the clarification.  I'll fix it as I have free time and maybe I'll add the most requested parameters.